src/Logic.hs

```diff
@@ -689,7 +708,7 @@ parseMergeCommand projectConfig triggerConfig = cvtParseResult . P.parse pCommen
     -- Parses the optional @ on friday@ command suffix. Since this starts with a
     -- space, it's important that the last run parser has not yet consumed it.
     pMergeWindow :: Parser MergeWindow
-    pMergeWindow = (OnFriday <$ P.try (P.hspace1 *> pString "on friday")) <|> pure NotFriday
+    pMergeWindow = (OnFriday <$ P.try (P.hspace *> pString "on friday")) <|> pure NotFriday
```

Comment:
It's Saturday and I just happened to see this on my GitHub feed. :see_no_evil: 
But before I look over this when I review the PR next week, these `hspace1` to `hspace` changes are not correct. The comments also explicitly mention these parsers having to consume a space character first. With your parser changes you can now `merge and deployto productionon friday`.